### DISCUSSION TITLE
iOS photo no EXIF metadata extracted

### DISCUSSION THREAD

**Discussion by ChenchaoZhao** [photoprism/photoprism#1418] [Q&A] [answered]
Hello

I used docker-compose to run the arm version of the awesome photoprism.

Then i tried to upload some photos from my iOS device.

I can see the images but none of them got metadata.

Thanks

**Comment by lastzero** [comment, marked as answer]
Check your phone settings and google the issue. Depending on the settings, your iPhone MAY remove metadata from uploads and/or convert the format from HEIC to JPEG.
